packages/dx-react-chart-demos/src/demo-sources/point-chart/scatter.jsxt

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';<%&additionalImports%>
+import {
+  Chart,
+  ScatterSeries,
+  ArgumentAxis,
+  ValueAxis,
+  Grid,
+} from '@devexpress/dx-react-chart-<%&themeName%>';
+import { Scale } from '@devexpress/dx-react-chart';
+
+const getRandom = () => {
+  const randomValue = Math.random();
+  if (randomValue === 0) {
+    return getRandom();
+  }
+  return randomValue;
+};
+
+function normalDistribution() {
```

Comment:
Can we change this syntax to `const normalDistribution = () => { ... }`

Reply:
Yes, we can